config/develop/airflow-ec2.yaml

```diff
@@ -0,0 +1,6 @@
+template_path: airflow-ec2.j2
+stack_name: airflow-ec2
+parameters:
+  VpcSubnet: !stack_output_external MWAA-VPC::PrivateSubnet1
```

Comment:
Nit: I'm not a huge fan of this MWAA VPC, so we would need a Jira ticket to create a VPC (if we don't already have a Jira ticket for it).

I don't see the MWAA VPC in this template, did you manually deploy it?

Reply:
I will create a ticket. It already exists in `dnt-dev` from our MWAA adventures and we let MWAA create one for us in `dpe-prod` to use in the interim